app/routes/users/components/UserProfile.js

```diff
@@ -47,20 +49,49 @@ type UpcomingEventsProps = {
   upcomingEvents: Array<Event>
 };
 
-const GroupPill = ({ group }: { group: Group }) => (
-  <Pill key={group.id} style={{ margin: '5px' }}>
-    {group.name}
-  </Pill>
-);
+const GroupPill = ({ group }: { group: Group }) =>
+  group.showBadge && (
+    <Pill key={group.id} style={{ margin: '5px' }}>
+      {group.name}
+    </Pill>
+  );
+
+const BadgeTooltip = ({
+  group,
+  start,
+  end
+}: {
+  group: Group,
+  start: Dateish,
+  end: ?Dateish
+}) => {
+  const startYear = moment(start).year();
+  const endYear = end ? moment(end).year() : 'd.d.';
+  return `${group.name} (${startYear} - ${endYear})`;
```

Comment:
@SmithPeder This looks a bit better :laughing:

Reply:
That looks a lot better 🥇